src/AppBundle/Tiqr/TiqrFactory.php

```diff
@@ -18,32 +18,48 @@
 namespace AppBundle\Tiqr;
 
 use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Tiqr_Service;
 use Tiqr_UserStorage;
 
-class TiqrServiceFactory
+class TiqrFactory
```

Comment:
Thanks for encapsulating the legacy :) Maybe add some more written documentation in this class.